kyaml/setters2/settersutil/fieldsetter.go

```diff
@@ -50,7 +50,7 @@ func (fs *FieldSetter) Filter(input []*yaml.RNode) ([]*yaml.RNode, error) {
 }
 
 // Set updates the OpenAPI definitions and resources with the new setter value
-func (fs FieldSetter) Set() (int, error) {
+func (fs *FieldSetter) Set() (int, error) {
```

Comment:
This is because of the attempt to assign at L88: `fs.SettersSchema = sc` does nothing on a non-pointer receiver